tests/unit/amo/components/TestRatingsByStar.js

```diff
@@ -146,61 +153,66 @@ describe(__filename, () => {
       1: 0,
     };
     const addon = addonForGrouping(grouping);
-    store.dispatch(setGroupedRatings({ addonId: addon.id, grouping }));
-    const root = render({ addon });
-    const counts = root.find('.RatingsByStar-star').find(Link);
-    const bars = root.find('.RatingsByStar-barContainer').find(Link);
+    const clientApp = CLIENT_APP_FIREFOX;
+    const lang = 'en-US';
 
-    function validateLink(link, score, expectedTitle) {
-      expect(link).toHaveProp(
-        'to',
-        reviewListURL({ addonSlug: addon.slug, score }),
-      );
-      expect(link).toHaveProp('title', expectedTitle);
-    }
+    const customStore = dispatchClientMetadata({ clientApp, lang }).store;
```

Comment:
You don't need to create a new `customStore` here. `dispatchClientMetadata` accepts a `store` argument, so you could just change this to `dispatchClientMetadata({ clientApp, lang, store })` and then use `store.dispatch` in the next line.

Reply:
Thanks! I will update that in next update.